types/sequelize/index.d.ts

```diff
@@ -3330,7 +3331,7 @@ declare namespace sequelize {
          * first element is the column / function to order by, the second is the direction. For example:
          * `order: [['name', 'DESC']]`. In this way the column will be escaped, but the direction will not.
          */
-        order?: string | col | literal | Array<string | number | Model<any, any> | { model: Model<any, any>, as?: string }> | Array<string | col | literal | Array<string | number | Model<any, any> | { model: Model<any, any>, as?: string }>>;
```

Comment:
I think this might be easier to read if you extracted a type for `Array<string | number | Model<any, any> | { model: Model<any, any>, as?: string } | fn>`.  Also, looks like there might be a couple of missing spaces before `|`s.

Reply:
Thank you for your comments.
I've created FindOptionsOrderArray type and fixed missing spaces.